src/utils/rest-api.js

```diff
@@ -484,6 +484,25 @@ function fetchEquipmentInfos(
     return backendFetch(fetchEquipmentInfosUrl);
 }
 
+export function fetchEquipmentExists(
```

Comment:
Need to add a head method  to avoid to get all equipment infos at each iteration ?
Need just to know if it exists

Reply:
it already exists that's what I used

Reply:
there is fetchEquipment and fetchEquipmentS